Error while building Geos Release 3.13.0 in Visual Studio for platform toolset "Visual Studio 2017(v141)"

Following error while building geos Release 3.13.0 in Visual Studio for platform toolset "Visual Studio 2017(v141)"
 
` [exec] C:\geos-3.13.0\dist\include\geos/operation/relateng/TopologyPredicate.h(197): error C2679: binary '<<': no operator found which takes a right-hand operand of type 'std::string' (or there is no acceptable conversion) `
   
   This seems to build OK when platform toolset "v143"
   
   This file `TopologyPredicate.h` seems to be introduced in latest release only

--- a/include/geos/operation/relateng/TopologyPredicate.h
+++ b/include/geos/operation/relateng/TopologyPredicate.h
@@ -18,6 +18,7 @@
 #include <geos/geom/Location.h>
 #include <geos/export.h>
 
+#include <ostream> // for operator<<
 
 // Forward declarations
 namespace geos {

